src/plugins/plot/configuration/PlotSeries.js

```diff
@@ -251,7 +257,12 @@ export default class PlotSeries extends Model {
             return this.limitEvaluator.evaluate(datum, valueMetadata);
         }.bind(this);
         const format = this.formats[newKey];
-        this.getYVal = format.parse.bind(format);
+        this.getYVal = (value) => {
+            const scale = 1; // TODO get from UI, positive number above 0
```

Comment:
What does `scale` do here? Can you please add a comment explaining what it's supposed to do?

Reply:
done